src/SFA.DAS.ApplyService.Database/Script.PostDeployment1.sql

```diff
@@ -32,108 +32,9 @@ BEGIN
-END
-
-IF NOT EXISTS (SELECT * FROM WhitelistedProviders  WHERE [UKPRN] = 10052982)
-BEGIN
-	INSERT INTO WhitelistedProviders ([UKPRN]) VALUES (10052982)
-END
-
-IF NOT EXISTS (SELECT * FROM WhitelistedProviders  WHERE [UKPRN] = 10060147)
-BEGIN
-	INSERT INTO WhitelistedProviders ([UKPRN]) VALUES (10060147)
-END
-
-IF NOT EXISTS (SELECT * FROM WhitelistedProviders  WHERE [UKPRN] = 10062192)
-BEGIN
-	INSERT INTO WhitelistedProviders ([UKPRN]) VALUES (10062192)
-END
-
-IF NOT EXISTS (SELECT * FROM WhitelistedProviders  WHERE [UKPRN] = 10064702)
-BEGIN
-	INSERT INTO WhitelistedProviders ([UKPRN]) VALUES (10064702)
-END
-
-IF NOT EXISTS (SELECT * FROM WhitelistedProviders  WHERE [UKPRN] = 10081936)
-BEGIN
-	INSERT INTO WhitelistedProviders ([UKPRN]) VALUES (10081936)
-END
-
-IF NOT EXISTS (SELECT * FROM WhitelistedProviders  WHERE [UKPRN] = 10082572)
-BEGIN
-	INSERT INTO WhitelistedProviders ([UKPRN]) VALUES (10082572)
-END
-
-IF NOT EXISTS (SELECT * FROM WhitelistedProviders  WHERE [UKPRN] = 10083239)
-BEGIN
-	INSERT INTO WhitelistedProviders ([UKPRN]) VALUES (10083239)
-END
-
-IF NOT EXISTS (SELECT * FROM WhitelistedProviders  WHERE [UKPRN] = 10084242)
-BEGIN
-	INSERT INTO WhitelistedProviders ([UKPRN]) VALUES (10084242)
-END
-
-IF NOT EXISTS (SELECT * FROM WhitelistedProviders  WHERE [UKPRN] = 10084697)
-BEGIN
-	INSERT INTO WhitelistedProviders ([UKPRN]) VALUES (10084697)
-END
-
-IF NOT EXISTS (SELECT * FROM WhitelistedProviders  WHERE [UKPRN] = 10084763)
-BEGIN
-	INSERT INTO WhitelistedProviders ([UKPRN]) VALUES (10084763)
-END
-
-IF NOT EXISTS (SELECT * FROM WhitelistedProviders  WHERE [UKPRN] = 10084763)
-BEGIN
-	INSERT INTO WhitelistedProviders ([UKPRN]) VALUES (10084763)
-END
-
-IF NOT EXISTS (SELECT * FROM WhitelistedProviders  WHERE [UKPRN] = 10084903)
-BEGIN
-	INSERT INTO WhitelistedProviders ([UKPRN]) VALUES (10084903)
-END
-
-IF NOT EXISTS (SELECT * FROM WhitelistedProviders  WHERE [UKPRN] = 10085115)
-BEGIN
-	INSERT INTO WhitelistedProviders ([UKPRN]) VALUES (10085115)
-END
-
-IF NOT EXISTS (SELECT * FROM WhitelistedProviders  WHERE [UKPRN] = 10029571)
-BEGIN
-	INSERT INTO WhitelistedProviders ([UKPRN]) VALUES (10029571)
-END
-
-IF NOT EXISTS (SELECT * FROM WhitelistedProviders  WHERE [UKPRN] = 10033656)
-BEGIN
-	INSERT INTO WhitelistedProviders ([UKPRN]) VALUES (10033656)
-END
-
-IF NOT EXISTS (SELECT * FROM WhitelistedProviders  WHERE [UKPRN] = 10016231)
-BEGIN
-	INSERT INTO WhitelistedProviders ([UKPRN]) VALUES (10016231)
-END
-
-IF NOT EXISTS (SELECT * FROM WhitelistedProviders  WHERE [UKPRN] = 10055520)
-BEGIN
-	INSERT INTO WhitelistedProviders ([UKPRN]) VALUES (10055520)
-END
-
-IF NOT EXISTS (SELECT * FROM WhitelistedProviders  WHERE [UKPRN] = 10063652)
-BEGIN
-	INSERT INTO WhitelistedProviders ([UKPRN]) VALUES (10063652)
-END
-
-IF NOT EXISTS (SELECT * FROM WhitelistedProviders  WHERE [UKPRN] = 10066450)
-BEGIN
-	INSERT INTO WhitelistedProviders ([UKPRN]) VALUES (10066450)
+	DELETE FROM WhitelistedProviders
```

Comment:
A better approach might be to have just this command:

TRUNCATE TABLE WhitelistedProviders